### PR TITLE
Optimize suggestions from other locales

### PR DESCRIPTION
**Pull request by mathjazz**
@Osmose @jotes r?

### CODE REVIEW

File: pontoon/base/views.py [hunk 1]
```diff
@@ -396,44 +396,28 @@ def get_translations_from_other_locales(request):
...
-    else:
-        return HttpResponse(
-            json.dumps(payload, indent=4), content_type='application/json')
+    payload = [{
+        'locale': {
+            'code': t.locale.code,
+            'name': t.locale.name
+        },
+        'translation': t.string
+    } for t in translations]
```
Comment (jotes) [inline pontoon/base/views.py]: we could change this into values() at some pont.

Comment (mathjazz) [inline pontoon/base/views.py]: Yeah, I tried that, but it's not possible to serialize values() as json directly.

Comment (mathjazz) [inline pontoon/base/views.py]: Thanks for the list() tip on IRC! This is much cleaner now.

File: pontoon/base/views.py [hunk 2]
```diff
@@ -396,44 +396,21 @@ def get_translations_from_other_locales(request):
...
-                "translation": translation.string
-            })
-
-    if len(payload) == 0:
-        return HttpResponse("error")
-    else:
-        return HttpResponse(
-            json.dumps(payload, indent=4), content_type='application/json')
+    payload = list(translations.values('locale__code', 'locale__name', 'string'))
+    return HttpResponse(json.dumps(payload, indent=4), content_type='application/json')
```
Comment (Osmose) [inline pontoon/base/views.py]: The indent seems unnecessary here.

**Comment by Osmose** [comment]
Nice fix!

**Pull request merged**